data/json/items/tools.json

```diff
@@ -5610,7 +5610,31 @@
     "category" : "weapons",
     "use_action": { "type": "knife" }
   },
-
+  {
+    "id": "glaive",
```

Comment:
True, was the reason I wound up not giving it the use_action if I recall. Either that, or possibly the item I based it off of didn't have that at the time.